### DISCUSSION TITLE
锄大地不灵了？

### DISCUSSION THREAD

**Discussion by FrankXu999** [moesnow/March7thAssistant#402] [Q&A]
更新了2.70之后，锄大地就开始不断地在模拟宇宙那里打开关闭地图，但是就是不开始工作。这是一个bug吗？还是我操作有问题？今天之前一直都好好的，更新到了最新的版本，各种设置都没有变

**Comment by moesnow** [comment]
大版本更新后UI有变化不是常事吗？主页→锄大地→更新锄大地，更新完成后即可正常使用。
